packages/vue/src/index.js

```diff
@@ -0,0 +1,68 @@
+import Vue from "vue";
+import microlink from "@microlink/vanilla";
+
+const camelCase = string =>
+  string.replace(/-([a-z])/g, g => g[1].toUpperCase());
+
+const createAnchor = url => {
+  const anchor = document.createElement("a");
+  anchor.href = url;
+  anchor.innerHTML = url;
+
+  return anchor;
+};
+
+export const Microlink = Vue.component("Microlink", {
+  props: {
+    url: {
+      type: String,
+      required: true
+    }
+  },
+
+  mounted() {
+    this.renderCard();
+  },
+
+  methods: {
+    renderCard() {
+      const { cardSpace } = this.$refs;
+      const anchor = createAnchor(this.url);
+
+      microlink(anchor, this.parameters, cardSpace);
```

Comment:
@Kikobeats passing a node to the new 3rd parameter allows vanilla to use that div as the "React base", and means we can also send a node to the 1st parameter without writing it to the document.